db/seeds/20180108175137-admin-user.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+require('dotenv').config();
+var bcrypt = require('bcrypt');
+var username = process.env.SUPERUSER_NAME || 'superuser';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    var password = process.env.SUPERUSER_PASSWORD;
+    var now = new Date().toISOString();
+
+    function createUser(encrypted_password) {
+      return queryInterface.bulkInsert('users', [{
+        name: username,
+        encrypted_password: encrypted_password,
+        superuser: true,
+        email: process.env.SUPERUSER_EMAIL,
+        created_at: now,
+        updated_at: now
+      }], {});
+    };
+
+    if (password) {
+      return bcrypt.hash(password, 10).then((encrypted_password) => createUser(encrypted_password));
```

Comment:
This can be simplified to `return bcrypt.hash(password, 10).then(createUser)`. LMK if you need explanation.